docs/game_sdk/Activities.md

```diff
@@ -160,6 +160,7 @@ activityManager.RegisterSteam(1938123);
 ## UpdateActivity
 
 Sets a user's presence in Discord to a new activity. This has a rate limit of 5 updates per 20 seconds.
+Bear in mind that it's possible for users to hide their currently played game (User Settings -> Game Activity).
```

Comment:
Can we reword this to exclude use of an idiom? This may make sense to users in some regions, but potentially not all. This also feels like something we should more directly callout with an info box. Maybe something like

```
>info
>It is possible for users to hide their presence on Discord (User Settings -> Game Activity). Presence set through this SDK may not be visible when this setting is toggled off.
```